src/search3_util.erl

```diff
@@ -1,12 +1,12 @@
 -module(search3_util).
 
--export([design_doc_to_index/2, get_doc/2]).
+-export([design_doc_to_index/3, get_doc/2]).
 
 -include_lib("couch/include/couch_db.hrl").
 
 -include("search3.hrl").
 
-design_doc_to_index(#doc{id=Id,body={Fields}}, IndexName) ->
+design_doc_to_index(#{db_prefix := DbPrefix}, #doc{id=Id,body={Fields}}, IndexName) ->
```

Comment:
ahhh I see, the main change here is that we're using `DbPrefix` which is already encoded, whereas previously I was just using the `DbName`